src/runtime/hvx_128.ll

```diff
@@ -286,4 +286,15 @@ define weak_odr <128 x i8> @halide.hexagon.shr.vb.vb(<128 x i8> %a, <128 x i8> %
   %r_32 = tail call <32 x i32> @llvm.hexagon.V6.vshuffeb.128B(<32 x i32> %sw_hi, <32 x i32> %sw_lo)
   %r = bitcast <32 x i32> %r_32 to <128 x i8>
   ret <128 x i8> %r
-}
\ No newline at end of file
+}
+
+define weak_odr void @halide.hexagon.l2fetch.Rt(i8* nocapture readonly %addr, i32 %desc) nounwind uwtable readnone alwaysinline {
```

Comment:
Yes these can be removed...they were used in the beginning before the runtime grew in complexity.